src/app/results/results.component.html

```diff
@@ -105,9 +105,6 @@
               </ul>
             </div>
           </div>
-            <div class="result message-bar" *ngIf="totalNumber > 0">
-                {{message}}
-            </div>
```

Comment:
@harshit98 I didnt't remove from videos section, I just made it common for all sections to avoid repetition of code